src/blob_file_set.cc

```diff
@@ -329,19 +329,36 @@ void BlobFileSet::GetObsoleteFiles(std::vector<std::string>* obsolete_files,
   obsolete_manifests_.clear();
 }
 
-void BlobFileSet::GetAllFiles(std::vector<std::string>* files, uint64_t* manifest_file_size) {
-  for (auto it = column_families_.begin(); it != column_families_.end(); ++it) {
-    auto& blob_storage = it->second;
-    // Get live blob file
-    blob_storage->GetAllFiles(files);
+void BlobFileSet::GetAllFiles(std::vector<std::string>* files, std::vector<VersionEdit>* edits) {
+  std::vector<std::string> all_blob_files;
+  
+  edits->clear();
+  edits->reserve(column_families_.size());
+  
+  for (auto& cf : column_families_) {
+    VersionEdit edit;
+    auto& blob_storage = cf.second;
+    // Get all blob files
+    blob_storage->GetAllFiles(&all_blob_files);
+    // Add all blob files to version_edit
+    edit.SetColumnFamilyID(cf.first);
+    edit.SetNextFileNumber(next_file_number_.load());
```

Comment:
no need to set next file number everytime

Reply:
Ok,  I will set next file number in first time.